src/pages/runs/[date].tsx

```diff
@@ -0,0 +1,81 @@
+import Layout from "@/components/Layout";
+import { Button } from "@/components/ui/button";
+import {
+  Card,
+  CardContent,
+  CardFooter,
+  CardHeader,
+  CardTitle,
+} from "@/components/ui/card";
+import { isMatch } from "date-fns";
+import { graphql, Link, PageProps } from "gatsby";
+import * as React from "react";
+
+const RunsPage: React.FC<PageProps<Queries.RunsPageQuery>> = ({
+  data,
+  params,
+}) => {
+  const drpdPages = data.allDrpdJson.edges;
+  const date = params.date;
+
+  return (
+    <Layout date={date}>
+      {isMatch(date, "yyyy-MM-dd") ? (
```

Comment:
Can we just do `{isMatch(date, "yyyy-MM-dd") && (`? Then we don't need the else condition of rendering an empty div.

Reply:
I think I wasn't sure of what to do for invalid date formats and just forgot about it, oops

Reply:
made it so that invalid dates redirect to home page